shaders/src/post_process.fs

```diff
@@ -5,5 +5,9 @@ void main() {
     // Invoke user code
     postProcess(inputs);
 
+#if defined(TARGET_MOBILE)
+    inputs.color = clamp(inputs.color, 0.0, MEDIUMP_FLT_MAX);
```

Comment:
We have a function called saturateMediump() for this you should add a vec3 variant. It's in common_math.fs I think.